lib/build-utilities/migrate-initializers.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const path = require('path');
+const existsSync = require('exists-sync');
+const fs = require('fs-extra');
+
+const fastbootInitializerTypes = [ 'initializers', 'instance-initializers'];
+const FASTBOOT_DIR = 'fastboot';
+
+/**
+ * Helper function to check if there are any `(instance-)?intializers/[browser|fastboot]/` path under the
+ * given root path.
+ *
+ * @param {String} rootPath
+ * @param {String} type
+ * @returns {Boolean} true if path exists
+ */
+function _checkInitializerTypeExists(rootPath, type) {
+  const isTypeExists = fastbootInitializerTypes.some((fastbootInitializerType) => {
+    const pathType = path.join(rootPath, 'app', fastbootInitializerType, type);
+
+    return existsSync(pathType);
+  });
+
+  return isTypeExists;
+}
+/**
+ * Helper function to check if there are any `(instance-)?intializers/browser/` path under the
+ * given root path and throw an error
+ *
+ * @param {String} rootPath
+ */
+function _checkBrowserInitializers(rootPath) {
+  const isBrowserInitializersPresent = _checkInitializerTypeExists(rootPath, 'browser');
+
+  if (isBrowserInitializersPresent) {
+    var errorMsg = `FastBoot build no longer supports ${rootPath}/app/(instance-)?initializers/browser structure. ` +
+                         `Please refer to www.ember-fastboot.com for a migration path.`;
```

Comment:
@kratiahuja I have picked up migrating an addon as we have discussed on Slack, and after updating `ember-cli-fastboot` to the latest version of your PR, I immediately ran into this error message, being caused by some other (unknown) addon. 

The problem I see here is that a) probably every existing app will run into this error once trying to migrate to FastBoot 1.0, and b) it gives no clue where that offending initializer actually comes from, which could be quite frustrating. It could be even some nested addon dependency the user does not see in his `package.json`. 

I think it would be cool (in this PR or a follow up) to have some better tracing available, like which addon is causing this, or at least what the initializers actual name is, to be able to effectively investigate into this!